src/frontends/common/include/openvino/frontend/node_context.hpp

```diff
@@ -0,0 +1,110 @@
+// Copyright (C) 2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <execinfo.h>
+#include <signal.h>
+#include <stdlib.h>
+#include <unistd.h>
```

Comment:
I suppose those are system dependent headers, why do we need them in public hpp?